solutions/swb-reference/src/SWBStack.ts

```diff
@@ -61,7 +60,8 @@ export class SWBStack extends Stack {
     CLIENT_ID: string;
     CLIENT_SECRET: string;
     USER_POOL_ID: string;
-    MAIN_ACCT_ENCRYPTION_KEY_ARN_OUTPUT_KEY: string;
+    S3_DATASETS_ENCRYPTION_KEY_ARN_OUTPUT_KEY: string;
+    S3_ARTIFACT_ENCRYPTION_KEY_ARN_OUTPUT_KEY: string;
```

Comment:
Do we need to output the S3 Artifact bucket key?

Reply:
we need it for provision environment. The environment-files parameter that gets passed during env provisioning is in the artifacts bucket